redundant-bmc/src/manager.cpp

```diff
@@ -80,26 +80,42 @@ sdbusplus::async::task<> Manager::doHeartBeat()
 
 Role Manager::determineRole()
 {
-    Role role{Role::Unknown};
+    using namespace role_determination;
+
+    RoleInfo roleInfo{Role::Unknown, ErrorCase::noError};
 
     try
     {
+        // Note:  If these returned nullopts, the algorithm wouldn't use
+        //        them anyway because there would be no heartbeat.
+        auto siblingRole = sibling->getRole().value_or(Role::Unknown);
+        auto siblingProvisioned = sibling->getProvisioned().value_or(true);
```

Comment:
Why are we treating the default state as 'Provisioned' for the sibling BMC?

How do we determine if the sibling BMC is not present, not detected, or has timed out, in order to switch to non-redundant mode?

Reply:
It won't matter much because we'll know from the lack of the sibling heartbeat that the sibling isn't alive, but I can change the default to false.

Reply:
done